FinancialTracker/Models/Transfer.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Transfer.swift
+//  FinancialTracker
+//
+//  Created by Boyan Pavlov on 16.02.22.
+//
+
+import Foundation
+
+enum TransferType: String, Codable {
+    case send = "Send"
+    case requestFromMe = "Request From Me"
+    case requestToMe = "Request To Me"
+    case receive = "Receive"
+}
+
+enum TransferState: String, Codable {
+    case pending = "Pending"
+    case completed = "Completed"
+}
+
+struct Transfer: Codable, Equatable {
+    var uid: String
+    var transferType: TransferType
+    var transferState: TransferState
+    var fromUser: String
+    var toUser: String
+    var amount: Double
+    var senderName: String
+    var senderCurrencyRate: Double
+    var receiverCurrencyRate: Double
+    var date: String
+    
+    enum TransferKeys: String {
```

Comment:
Also, I don't think you need to declare all `TransferKeys` - looks like you are using just the `receiverCurrencyRate` and the `transferState` keys.